WalletWasabi.Fluent/Views/Receive/ReceiveView.axaml

```diff
@@ -0,0 +1,41 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:c="using:WalletWasabi.Fluent.Controls"
+             xmlns:receive="using:WalletWasabi.Fluent.ViewModels.Receive"
+             xmlns:behaviors="using:WalletWasabi.Fluent.Behaviors"
+             xmlns:i="using:Avalonia.Xaml.Interactivity"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="receive:ReceiveViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Receive.ReceiveView">
+
+  <c:ContentArea Title="{Binding Title}"
+                 EnableNext="True" NextContent="Continue"
+                 EnableCancel="True">
+    <DockPanel>
+      <StackPanel DockPanel.Dock="Top" HorizontalAlignment="Stretch">
+        <Label Content="_Reference" Target="ReferenceTbx" HorizontalAlignment="Left"/>
```

Comment:
The Reference latent and ContentArea title are both bold, make we can me this text bit lighter.

Reply:
Good point, fixed